packages/components/forma-36-react-timepicker/package.json

```diff
@@ -33,8 +33,8 @@
     "@types/is-hotkey": "^0.1.1",
     "@types/jest": "^24.0.22",
     "@types/lodash.orderby": "^4.6.6",
-    "@types/react": "^16.9.11",
-    "@types/react-dom": "^16.9.3",
+    "@types/react": "^16.9.56",
+    "@types/react-dom": "^16.9.9",
```

Comment:
do you think we should add `csstype` as a dependency, so we control the version and avoid relying on it from react?

Reply:
Yeah good point, we should when we're using it directly anyway.